react-components/stories/Reveal3DResources.stories.tsx

```diff
@@ -22,13 +23,20 @@ export const Main: Story = {
   args: {
     resources: [
       {
-        modelId: 1791160622840317,
-        revisionId: 498427137020189,
-        transform: new Matrix4().makeTranslation(40, 0, 0)
+        modelId: 3544114490298106,
+        revisionId: 6405404576933316,
+        transform: new Matrix4().makeTranslation(40, 0, 0),
+        styling: {
+          nodeGroups: [
+            { treeIndexSet: new IndexSet([2, 4, 6, 8]), style: { color: new Color('blue') } },
+            { nodeIds: [7152264879809192, 7899489067916664], style: { visible: false } }
```

Comment:
Should we allow both node ids and treeindex sets in node groups, or should we encourage the user to transform from nodeids to treeindex sets for usage with low end apis